.github/workflows/twitter.yml

```diff
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+name: Twitter, together!
+jobs:
+  preview:
+    name: Preview
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: twitter-together/action@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  tweet:
+    name: Tweet
+    runs-on: ubuntu-latest
+    environment: is-php-dying
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

Comment:
Haven't had time to read `twitter-together` in detail yet, so just for clarification:

If we create a preview of a tweet using `Preview` job, and after that there is any other non-PR related push, the tweet will not be published?

Anyway PR looks good, thanks a lot!

Reply:
As I understand, the logic of getting new tweets is independent between `pull_request` and `push` events so it should work:
https://github.com/twitter-together/action/blob/main/lib/push/get-new-tweets.js
https://github.com/twitter-together/action/blob/main/lib/pull-request/get-new-tweets.js